backend/keepalived/keepalived.go

```diff
@@ -480,6 +480,7 @@ func (k *Keepalived) dead(key string, prev liveness.State, leader bool) bool {
 	if entity == nil {
 		// The entity has been deleted, there is no longer a need to
 		// track keepalives for it.
+		logger.Debug("nil entity")
```

Comment:
Minor. Should this be using `lager` so it includes the additional context?

Reply:
Yes, thanks!